pipeline/transformer/transformer_tei/src/main/scala/weco/pipeline/transformer/tei/transformers/TeiContributors.scala

```diff
@@ -0,0 +1,276 @@
+package weco.pipeline.transformer.tei.transformers
+
+import cats.instances.either._
+import cats.syntax.traverse._
+import weco.catalogue.internal_model.identifiers.IdState.{
+  Identifiable,
+  Unminted
+}
+import weco.catalogue.internal_model.identifiers.{
+  IdState,
+  IdentifierType,
+  SourceIdentifier
+}
+import weco.catalogue.internal_model.work.{
+  ContributionRole,
+  Contributor,
+  Person
+}
+import weco.pipeline.transformer.result.Result
+
+import scala.xml.{Elem, Node, Text, XML}
+
+object TeiContributors {
+
+  /**
+    * author nodes appear only within msItem. They contain the name of the Author of the work and optionally the id of the Author.
+    * The Id of the Author refers to two different authorities depending on whether the work is an Arabic manuscript or not
+    */
+  def authors(
+    node: Node,
+    isFihrist: Boolean
+  ): Result[List[Contributor[IdState.Unminted]]] =
+    (node \ "author")
+      .map { n =>
+        for {
+          authorInfo <- getLabelAndId(n)
+          (label, id) = authorInfo
+          res <- createContributor(
+            label = label,
+            id = id,
+            isFihrist = isFihrist,
+            ContributionRole("author"))
+        } yield res
+
+      }
+      .toList
+      .sequence
```

Comment:
Copied unmodified from TeiAuthors.

Reply:
TeiAuthors was renamed to TeiContributors but the changes were so many that gihub didn't figure it out

Reply:
Ah, sorry, half of these comments are for my benefit as I was trying to read the diff.